### DISCUSSION TITLE
Integrate aspect-ratio into tailwindcss

### DISCUSSION THREAD

**Discussion by cperrin88** [tailwindlabs/tailwindcss#6281] [Ideas]
I have looked into the aspect-ratio CSS property and was surprised to see that [aspect-ratio is now supported by all recent browsers](https://caniuse.com/mdn-css_properties_aspect-ratio). The aspect-ratio plugin says that it should be integrated into tailwind when this is the case.

That is why I want to propose just that. Integrate aspect-ratio into tailwindcss.

**Comment by adamwathan** [comment]
Already done! https://github.com/tailwindlabs/tailwindcss/pull/5359

**Reply by cperrin88** [reply]
Oh. You guys are sneaky. 🤣
